modin/pandas/dataframe.py

```diff
@@ -991,9 +991,11 @@ def insert(
         """
         Insert column into ``DataFrame`` at specified location.
         """
+        from modin.numpy import array
```

Comment:
cannot we import it at the top of the module?

Reply:
No, because we have `import modin.pandas as pd` in `modin.numpy.arr.py`.

We can make an abstract type and use it for type checks instead. Something like `ABCModinArray`.